lesson16/css/contents.css

```diff
@@ -0,0 +1,47 @@
+.tab-content-box {
+  padding: 20px 40px;
+  border: 1px solid skyblue;
+}
+
+.tab-content {
+  display: none;
+  width: 100%;
+}
+
+.tab-content.show {
+  display: block;
+}
+
+.fa-bell {
+  display: none;
+}
+
+.fa-bell.new {
+  display: inline;
+}
+
+.picture-container img {
+  width: 200px;
+  height: 200px;
+  padding: 10px;
+}
```

Comment:
nits
I was concerned that the aspect ratio was different from the actual image. But CSS is not a must, so don't worry about it here.

Reply:
Thank you for the comment Mai-san,
the pictures have different ratios so I thought it probably a good idea to use the same ratio for all of them.
Please let me know if there is a better way to define picture size.

Reply:
I think we just need to create the original image with a 1:1 aspect ratio in advance.
Or, you can put an image in the background and use background-cover or not.
But that's not very important for this assignment, so you can ignore it!